sympy/solvers/ode/single.py

```diff
@@ -917,6 +917,93 @@ def _get_general_solution(self, *, simplify_flag: bool = True):
         return [gensol]
 
 
+class RationalRiccati(SinglePatternODESolver):
+    r"""
+    Gives all rational solutions to the first
+    order Riccati Differential Equation
+
+    .. math :: y' = b_0(x) + b_1(x) y + b_2(x) y^2
+
+    where `b_0`, `b_1` and `b_2` are rational functions of `x`
+    with `b_2 \ne 0` (`b_2 = 0` would make it a Bernoulli equation).
```

Comment:
If `b_0 = 0` it's Bernoulli.

Reply:
I think its a little ambiguous because Bernoulli can refer to equations of the form `y' + P*y = Q` or the more general form `y' + P*y = Q*y^n`. Maybe its just better to avoid mentioning this here.

Reply:
I think it's worth saying something here. There are two relevant points:

1. If either `b_0` or `b_2` is zero then the equation comes under a simpler class (Bernoulli or linear respectively) that can be solved more directly.
2. The algorithm here can find any rational solution for any equation of this form provided `b_2 != 0`.